max_h: 40
max_s: 255
max_v: 255
min_h: 20
min_s: 0
min_v: 0

# h=34
